py-polars/rust-toolchain

```diff
@@ -1 +1 @@
-nightly
+nightly-2021-07-06
```

Comment:
Can this be restored to nightly?

Reply:
I did this to fix my VSCode to work with Rust language server since not every nightly released with it, will change back to the original value when commit

Reply:
I cannot open my web browser or I read that rust analyzer is a looot better than RLS. I'd recomend using that. :slightly_smiling_face:

Reply:
Welp, time to switch, RLS consume way too much memory. have to turn that off each time I compile polars. :(

Reply:
Managed to get rust analyzer to work, less mem consumed, happier dev experience. Thank you.